docs/bot/usage.md

```diff
@@ -24,7 +24,7 @@ The bot will then create a Pull Request to add the contributor, then reply with
 > Your request to the bot doesn't need to be perfect. The bot will use [basic Natural Language Parsing](https://github.com/all-contributors/all-contributors-bot/blob/master/src/tasks/processIssueComment/utils/parse-comment/index.js) to determine your intent.
 > For example, this will work too:
 >
-> `Jane you are crushing it in documentation and your infrastructure work has been great too. Let's add jane.doe23 for her contributions. cc @all-contributors`
+> `Jane you are crushing it in documentation and your infrastructure work has been great too. Let's add jane.doe23 for her contributions. cc @all-contributors bot`
```

Comment:
This isn't actually mandatory since the bot responds to `@all-contributors` so this unnecessary.